js/server.js

```diff
@@ -6,12 +6,10 @@ const logger = require('./logger').getLogger('application');
 const app = express();
 const port = nconf.get('server:port');
 
-
 function start() {
-
-    app.get('/', (req, res) => {
-        res.send(200, { status: 'ok' });
-    });
+   
+    //Routes
+    app.use(require('./routes'))
```

Comment:
Would be good if you can import this file at top and then pass that variable here.